### PR TITLE
LF-2640 add helptext to abandon task screen informing that hours still count toward payroll

### PR DESCRIPTION
**Pull request by seinnlee**
Ticket [LF-2640](https://lite-farm.atlassian.net/browse/LF-2640)

To Test:

1. Create a task and assign it to someone
2. Click "Abandon this task"
3. There should be an info tooltip next to "Was any work completed for this task?"
4. When hovering over, an info text should appear (click when on mobile)

### CODE REVIEW

File: packages/webapp/src/components/Task/AbandonTask/index.jsx [hunk 1]
```diff
@@ -113,7 +115,21 @@ const PureAbandonTask = ({
...
+                  paddingRight: '10px',
+                  fontSize: '16px',
+                  lineHeight: '20px',
+                  display: 'inline-block',
+                }}
+              >
+                {<Infoi content={t('TASK.ABANDON_TASK_HELPTEXT')} />}
+              </Label>
+            </div>
+          </div>
```
Comment (AbdusShaikh) [inline packages/webapp/src/components/Task/AbandonTask/index.jsx:132]: This works but there is a much simpler way of doing the same thing.

The 'Main' component has a 'toolTipContent' prop. We can just pass t('TASK.ABANDON_TASK_HELPTEXT') into 'Main' as a prop and get the same result

File: packages/webapp/src/components/Task/AbandonTask/index.jsx [hunk 2]
```diff
@@ -113,7 +115,21 @@ const PureAbandonTask = ({
 
       {hasAssignee && (
         <>
-          <Main style={{ marginBottom: '24px' }}>{t('TASK.ABANDON_TASK_DURATION')}</Main>
```
Comment (AbdusShaikh) [inline packages/webapp/src/components/Task/AbandonTask/index.jsx:116]: We can make this:

`<Main style={{ marginBottom: '24px' }} tooltipContent={t('TASK.ABANDON_TASK_DURATION')}>{t('TASK.ABANDON_TASK_DURATION')}</Main>`

**Review by AbdusShaikh — CHANGES_REQUESTED** [review]
Works but there is a simpler and cleaner method of doing the same thing

File: packages/webapp/src/components/Task/AbandonTask/index.jsx (hunk 2 above)
Comment (seinnlee) [reply packages/webapp/src/components/Task/AbandonTask/index.jsx:116]: That is much simpler and cleaner. Thanks!

**Review by AbdusShaikh — APPROVED** [review]
Looks good!

**Review by Cali0707 — APPROVED** [review]
Looks good to me!

**Pull request merged**